CHALLENGE 2.0/def,multiple_3_calc.py

```diff
@@ -0,0 +1,8 @@
+def mul_calc(num):
+    num=float(input("enter value here; "))
+    arrayA=range(3,100,5)
+    for i in arrayA:
+        print(i%num==0)
+
```

Comment:
Your function does not return anything. I am expecting it to return the answer. Use `return` instead of `print`

Reply:
Now resolved